languages/js/src/Host.ts

```diff
@@ -158,6 +205,26 @@ export class Host {
     return instanceId;
   }
 
+  /**
+   * Register the MROs of all classes that are registered on `this`.
+   */
+  registerMros() {
+    // Get MRO of all registered classes
+    // NOTE: not ideal that the MRO gets updated each time loadStr is
+    // called, but since we are planning to move to only calling load once
+    // with the include feature, I think it's okay for now.
+    for (const typ of this.distinctUserTypes()) {
+      // Get MRO for type.
+      const mro = ancestors(typ.cls)
+        // TODO: as conversion ok?
+        .map(c => this.getType(c as Class)?.id)
+        .filter(id => !!id);
```

Comment:
Why is the `!!` necessary?

Reply:
not sure this ever matters (do we ever assign `id=0` for anything, or do we start at 1?) but this will filter out `id=0`. Maybe add an invariant comment somewhere (maybe on the `getType` definition?) that ID will always be a positive integer?

Oh and if we're sure we're never going to encounter `id=0`, you can remove the `!!` (`.filter(id => id)`) or use `.filter(Number)` if you prefer that

Reply:
Changed this to explicitly check that id is not undefined.